codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -273,7 +272,7 @@ public final void finishHandshake(Channel channel, FullHttpResponse response) {
             // Delay the removal of the decoder so the user can setup the pipeline if needed to handle
             // WebSocketFrame messages.
             // See https://github.com/netty/netty/issues/4533
-            channel.eventLoop().execute(new OneTimeTask() {
+            channel.eventLoop().execute(new Runnable() {
```

Comment:
I think it makes sense to remove `OneTimeTask` in this PR as its existence dependents upon `MpscLinkedQueueNode` which should be removed in this PR.